### PR TITLE
Updated the ide option to use the default open in browser

### PR DESCRIPTION
**Pull request by jeremyFreeAgent**
Related to https://github.com/symfony/symfony/pull/19973

### CODE REVIEW

**Comment by javiereguiluz** [comment]
I like this proposal because this is the only value that will work for everyone. Thanks @jeremyFreeAgent

**Comment by jeremyFreeAgent** [comment]
You should thank @nicolas-grekas because he is the one who told me there was the `ide` option set in the symfony-demo ;)

File: app/config/config.yml [hunk 1]
```diff
@@ -19,12 +19,11 @@ parameters:
 
 # Basic configuration for the Symfony framework features
 framework:
-    # Uncomment the 'ide' option to turn all of the file paths in an exception
-    # page into clickable links that open the given file using your favorite IDE.
-    # Supported values are 'textmate', 'macvim', 'emacs' and 'sublime' shortcuts
-    # and any custom configuration string, such as: "phpstorm://open?file=%%f&line=%%l"
+    # The 'ide' option turns all of the file paths in an exception page
+    # into clickable links that open the given file using your favorite IDE.
+    # Supported values are 'phpstorm', 'sublime', 'textmate', 'macvim' and 'emacs'.
```
Comment (fabpot) [inline app/config/config.yml]: You should probably explain the null value here.

**Comment by sargath** [comment]
Bare in mind that the user should have configured a link handler in his browser to open up those -
http://davidbu.ch/mann/blog/2014-05-06/open-php-stack-trace-links-phpstorm.html

Not every OS can handle this by its own.

**Comment by bocharsky-bw** [comment]
I'd like to promote PhpStorm here since it's cross-platform and it has a nice Symfony plugin. Obviously, it's simply the best IDE for PHP apps and especially for Symfony apps.

File: app/config/config.yml (hunk 1 above)
Comment (jeremyFreeAgent) [inline app/config/config.yml]: Done

Comment (jeremyFreeAgent) [inline app/config/config.yml]: Thanks :)

**Comment by javiereguiluz** [comment]
Let's merge this because Symfony 3.2 has already been released and using "null" in this option is now the default value. Thanks @jeremyFreeAgent!

**Pull request merged**